frame.go

```diff
@@ -173,11 +173,7 @@ func (fr *frameImpl) Labels() map[string]interface{} {
 
 // Len is the number of rows
 func (fr *frameImpl) Len() int {
-	if len(fr.columns) > 0 {
-		return fr.columns[0].Len()
-	}
-
-	return 0
+	return fr.indices[0].Len()
```

Comment:
do we always have something in indices that you can access the array without checking it's length?
what about empty df?

Reply:
Every dataframe has an index.

Reply:
Ok, turns out this can happen not because of pandas but because of how we populate the structure internally...